cnf-certification-test/accesscontrol/securitycontextcontainer/securitycontextcontainer.go

```diff
@@ -0,0 +1,454 @@
+package securitycontextcontainer
+
+// https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/
+// api we used as a reference
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+
+	"github.com/sirupsen/logrus"
+	"github.com/test-network-function/cnf-certification-test/pkg/provider"
+	"github.com/test-network-function/cnf-certification-test/pkg/stringhelper"
+	"github.com/test-network-function/cnf-certification-test/pkg/tnf"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	category1        = "category1"
+	haveRequiredDrop = "haveRequiredDrop"
+)
+
+type OkNok int
+
+const (
+	OKNOK = iota
+	NOK   // 0
+	OK    // 1
+)
+
+const (
+	OKString  = "true"
+	NOKString = "false"
+)
+
+// print the strings
+func (okNok OkNok) String() string {
+	switch okNok {
+	case OK:
+		return OKString
+	case NOK:
+		return NOKString
+	}
+	return "false"
+}
+
+type ContainerSCC struct {
+	HostDirVolumePluginPresent OkNok // 0 or 1 - 0 is false 1 - true
+	HostIPC                    OkNok
+	HostNetwork                OkNok
+	HostPID                    OkNok
+	HostPorts                  OkNok
+	PrivilegeEscalation        OkNok // this can be true or false
+	PrivilegedContainer        OkNok
+	RunAsUserPresent           OkNok // thes filed that checking if the value is present
+	ReadOnlyRootFilesystem     OkNok
+	RunAsNonRoot               OkNok
+	FsGroupPresent             OkNok
+	SeLinuxContextPresent      OkNok
+	Capabilities               string
+	HaveDropCapabilities       OkNok
+	AllVolumeAllowed           OkNok
+}
+
+var (
+	requiredDropCapabilities = []string{"MKNOD", "SETUID", "SETGID", "KILL"}
+	dropAll                  = []string{"ALL"}
+	category2AddCapabilities = []string{"NET_ADMIN, NET_RAW"}
+	category3AddCapabilities = []string{"NET_ADMIN, NET_RAW, IPC_LOCK"}
+	Category1                = ContainerSCC{NOK,
+		NOK,
+		NOK,
+		NOK,
+		NOK,
+		OK,
+		NOK,
+		OK,
+		NOK,
+		OK,
+		OK,
+		OK,
+		category1,
+		OK,
+		OK}
```

Comment:
Add comments to reference categories:
	Category1                = ContainerSCC{
		NOK, //HostDirVolumePluginPresent
		NOK, //HostIPC
		NOK, //HostNetwork
		NOK, //HostPID
		NOK, //HostPorts
		OK, //PrivilegeEscalation
		NOK, //PrivilegedContainer
		OK, //RunAsUserPresent
		NOK, //ReadOnlyRootFilesystem
		OK, //RunAsNonRoot
		OK, //FsGroupPresent
		OK, //SeLinuxContextPresent
		category1,//Capabilities
		OK, //HaveDropCapabilities
		OK} //AllVolumeAllowed